app/src/main/java/org/fossasia/openevent/app/login/LoginActivity.java

```diff
@@ -59,7 +66,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
         btnLogin.setOnClickListener(v -> {
 
-            String email = etEmail.getText().toString();
+            String email = etEmail.getEditText().getText().toString();
```

Comment:
Yes, to include the autocomplete text view. To get the test from the textinputlayout, edittext needs to be called first.